datastream-server-restli/src/main/java/com/linkedin/datastream/server/dms/DatastreamResources.java

```diff
@@ -102,7 +102,7 @@ public class DatastreamResources extends CollectionResourceTemplate<String, Data
   // To support retries on the request timeouts
   public static final String CONFIG_STOP_TRANSITION_TIMEOUT_MS = "stopTransitionTimeoutMs";
   public static final String CONFIG_STOP_TRANSITION_RETRY_PERIOD_MS = "stopTransitionRetryPeriodMs";
-  private static final Long STOP_TRANSITION_TIMEOUT_MS_DEFAULT = Duration.ofMillis(60000).toMillis();
+  private static final Long STOP_TRANSITION_TIMEOUT_MS_DEFAULT = Duration.ofMillis(90000).toMillis();
```

Comment:
discussion: Was the original value of the timeout based, whole or in part, on Nurse? My understanding is that part of motivation for this work was that the current `STOP`/`START`/`RESTART` process is unreliable given our automation will timeout after 60s. Can this timeout be lowered to 60s and any of its dependents lowered accordingly?

Reply:
This is the timeout of the rest.li node handling the stop request, i.e. how long will it wait for the leader to mark the datastream STOPPED. I don't think the choice of this was based on the Nurse, @shrinandthakkar can correct me here if I'm wrong.

The reason we decided to increase this is because now the leader also has a 60 second timeout to hear from followers that they stopped the stream. With this change we're making sure the leader's 60 seconds poll times out before the rest.li requests' timeout. Eventually, with Phase 2, this will be revisited.

Reply:
We need a way to set dependencies on the other timers in the code itself. The reasoning for 90 second related to another 60 second timeout, should also reflect in the code. 

for eg: TimeoutA= 60sec TimeoutB = TimeoutA + 30sec rather than TimeoutA=60sec and TimeoutB=30sec as former helps in understanding the dependency.

Also, please add comments.